### PR TITLE
Methods of RegExp and String

### PR DESCRIPTION
**Pull request by fsdude**
Translation to portuguese done. Followed the english version again.

Source file can be found: https://github.com/javascript-tutorial/en.javascript.info/blob/master/9-regular-expressions/17-regexp-methods/article.md

### CODE REVIEW

**Comment by odsantos** [comment]
Hi @fsdude  would you please resolve the conflicts, or kindly inform if unavailable to? Thank you.

**Comment by fsdude** [comment]
> Hi @fsdude would you please resolve the conflicts, or kindly inform if unavailable to? Thank you.

Hello! Yeah.. lately I've been busy with some work. I don't know if I'll be able to fix this soon, sorry for that =)
Thanks by the way

**Review by odsantos — CHANGES_REQUESTED** [review]
Hi @kiniggit, @luizclr, @adriavieira314,  @Mr-Saxobeat, please since @fsdude might be unavailable I would appreciate if any of you could review this PR #141 , including my revision.
Thank you.

**Comment by odsantos** [comment]
Hi @kiniggit, @luizclr, @adriavieira314, @Mr-Saxobeat, please since @fsdude might be unavailable I would appreciate if any of you could review this PR #141 , including my changes.
Thank you.

**Comment by Peruibeloko** [comment]
@odsantos aqui também tem o problema da tradução dos `pattern:`

**Comment by odsantos** [comment]
> @odsantos aqui também tem o problema da tradução dos `pattern:`

@Peruibeloko I'm afraid I do not know what you intend to say with this phrase. Please, could you share more details? Thanks.

**Comment by odsantos** [comment]
> @odsantos aqui também tem o problema da tradução dos `pattern:`

@Peruibeloko I'm afraid I do not know what you intend to say with this phrase. Please, could you share more details? Thanks.

**Comment by Peruibeloko** [comment]
@odsantos the PRs #147, #141 (this one) and #274 all translated parts of the text that weren't meant to have been translated.

The parts in question are the \`pattern:\`, \`subject:\` and \`match:\` decorators used in the Regex chapters, as they're used by the site builder literally. Changing this might break the builds! My evidence for this is that all of them match the class names used in the HTML output of the site.

File: 9-regular-expressions/17-regexp-methods/article.md
Comment (Peruibeloko) [inline 9-regular-expressions/17-regexp-methods/article.md:11]: The `pattern:` syntax shouldn't be translated

File: 9-regular-expressions/17-regexp-methods/article.md
Comment (Peruibeloko) [inline 9-regular-expressions/17-regexp-methods/article.md:27]: The `pattern:` syntax shouldn't be translated

File: 9-regular-expressions/17-regexp-methods/article.md
Comment (Peruibeloko) [inline 9-regular-expressions/17-regexp-methods/article.md:37]: The `pattern:` syntax shouldn't be translated

File: 9-regular-expressions/17-regexp-methods/article.md
Comment (Peruibeloko) [inline 9-regular-expressions/17-regexp-methods/article.md:58]: This shouldn't be translated

File: 9-regular-expressions/17-regexp-methods/article.md
Comment (Peruibeloko) [inline 9-regular-expressions/17-regexp-methods/article.md:138]: The `pattern:` syntax shouldn't be translated

File: 9-regular-expressions/17-regexp-methods/article.md
Comment (Peruibeloko) [inline 9-regular-expressions/17-regexp-methods/article.md:237]: The `pattern:` syntax shouldn't be translated

File: 9-regular-expressions/17-regexp-methods/article.md
Comment (Peruibeloko) [inline 9-regular-expressions/17-regexp-methods/article.md:239]: The `pattern:` syntax shouldn't be translated

File: 9-regular-expressions/17-regexp-methods/article.md
Comment (Peruibeloko) [inline 9-regular-expressions/17-regexp-methods/article.md:277]: The `pattern:` syntax shouldn't be translated

File: 9-regular-expressions/17-regexp-methods/article.md
Comment (Peruibeloko) [inline 9-regular-expressions/17-regexp-methods/article.md:279]: The `pattern:` syntax shouldn't be translated

File: 9-regular-expressions/17-regexp-methods/article.md
Comment (Peruibeloko) [inline 9-regular-expressions/17-regexp-methods/article.md:315]: The `pattern:` syntax shouldn't be translated

**Comment by odsantos** [comment]
> @odsantos the PRs #147, #141 (this one) and #274 all translated parts of the text that weren't meant to have been translated.
> 
> The parts in question are the `pattern:`, `subject:` and `match:` decorators used in the Regex chapters, as they're used by the site builder literally. Changing this might break the builds! My evidence for this is that all of them match the class names used in the HTML output of the site.

@Peruibeloko I'll include your corrections as soon as possible. Thank you.

**Comment by nazarepiedady** [comment]
@odsantos, @jonnathan-ls, @Peruibeloko, what do you recommend colleagues to be done in this pull request?

**Review by jonnathan-ls — APPROVED** [review]
@nazarepiedady @odsantos @Peruibeloko 

I updated the PR with the master branch, fixed merge conflicts, translated pending content and applied the suggestions highlighted as code review https://github.com/javascript-tutorial/pt.javascript.info/pull/141/commits/3565c179819894d7389c11ee4d346341f7477164

Can you evaluate this PR again?

I'm also leaving my LGTM as a review

File: 9-regular-expressions/17-regexp-methods/article.md
Comment (nazarepiedady) [inline 9-regular-expressions/17-regexp-methods/article.md:7]: @jonnathan-ls, thanks for saving some of your time to work on this. It is very praiseworthy.

I would like to point out a little spelling mistake where it is written `str.macth` should be `str.match`.

**Review by nazarepiedady — COMMENTED** [review]
@jonnathan-ls, is all pending work solved?

File: 9-regular-expressions/17-regexp-methods/article.md
Comment (jonnathan-ls) [reply 9-regular-expressions/17-regexp-methods/article.md]: Thanks for the observation, I added the correction

**Review by nazarepiedady — APPROVED** [review]
It was an excellent work, let's go forward.

**Comment by nazarepiedady** [comment]
@jonnathan-ls, you need to approve and merge if you have time for that, too.

**Comment by jonnathan-ls** [comment]
> you need to approve and merge if you have time for that, too.

I left my approval, but I don't see any point in merging without the other reviewers who registered observations not reviewing it again yet.

I don't think it's worth closing change requests from other code reviewers

**Comment by nazarepiedady** [comment]
@jonnathan-ls, we will wait most likely for centuries if we wait for others even when we have two approvals. The current pull request is 4 years old. It is too much time.

**Comment by nazarepiedady** [comment]
@jonnathan-ls, let's do this way. You approve, and we will wait at least one week to get one more approval to merge the changes.

**Comment by nazarepiedady** [comment]
@Peruibeloko, could you help us review this 4 years old pull request?

**Review by Peruibeloko — APPROVED** [review]

**Comment by Peruibeloko** [comment]
LGTM!

**Pull request merged**
